### DISCUSSION TITLE
Ability to Upload Multiple Backgrounds and Change Over Period of Time

### DISCUSSION THREAD

**Discussion by dukerwang** [bluecaret/carettab#378] [Ideas and Feature Requests]
Please add the ability to upload multiple images as backgrounds and allow the background to change randomly once every certain amount of time (the amount of time can be changed as an option).

**Comment by Colleenmp** [comment]
I agree. It would be nice to see my own background images in each new tab, especially if they could be accessed directly from Google Photos instead of from the limited storage space that my Chromebook and my Google Cloud Storage (MyDrive) have to offer.
